docs/operations/hotplug_interfaces.md

```diff
@@ -100,6 +100,23 @@ You can now check the VMI status for the presence of this new interface:
 kubectl get vmi vmi-fedora -ojsonpath="{ @.status.interfaces }"
 ```
 
+### Migration based hotplug
+In case your cluster doesn't meet the [requirements](#requirements), it's possible to hotplug an interface by migrating the VM.
+The actual attachment won't take place immediately, and the new interface will be available in the guest once the migration is completed.
+
+#### Add new interface:
```

Comment:
```suggestion
#### Hotplugging an interface
```

Reply:
1. Could you please change the heading to `###`?
2. nit: Could you please remove the colon `:`?